### DISCUSSION TITLE
Curious about the definition of MSforID distance

### DISCUSSION THREAD

**Discussion by issararab** [YuanyueLi/SpectralEntropy#7] [General]
Hello,
It would be awesome if you can explain what does the MSforID (version 1) distance compute?
I am not interested in the mathematical formula, but in word definition/explanation of this distance between a query and target spectrum. I can't find anything about it online, just a database name.
Regards,
AI

**Comment by YuanyueLi** [comment]
Hello,

It's adopted from this paper: https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/jms.1525

Best,
Yuanyue
